README.md

```diff
@@ -136,7 +136,8 @@ therefore want any module to be understandable to all, not only the `{admiral}`
 ## Conference Presentations
 
 * [R/Pharma 2021 talk](https://www.youtube.com/watch?v=N7Bw8c3D5fU) (recording)
-* [PHUSE EU Connect 2021 workshop](https://github.com/pharmaverse/admiral.phuse.workshop/tree/main/EU%202021) (slides and materials)
+* [PHUSE US Connect 2022 pharmaverse workshop](https://github.com/pharmaverse/pharmaverse.workshop.phuseUS2022) (slides and materials - including `{admiral}`
+workshop slides from PHUSE EU Connect 2021)
```

Comment:
Change to PHUSE US Connect 2022

Reply:
Line 140

Reply:
that section was to highlight that we also have the `admiral` EU slides there in addition to all the materials from the US `pharmaverse` workshop